From 966f66b3da9d15d429cbbc1dbc1466766306afdf Mon Sep 17 00:00:00 2001
From: Danny Baumann <dannybaumann@web.de>
Date: Wed, 18 Jul 2018 17:11:12 +0200
Subject: [PATCH 20/20] Don't re-show search bar on query click.

If we click query the search bar must be visible already. The actual
problem with this, however, is that this is triggered via the default
implementation of onKeyUp in the query view, which means that it's
triggered when pressing enter on the keyboard, but _after_ the submit
IME action is delivered.

Change-Id: I4b3d7ec3c0040a3ac7cf8460f8059f5074f9149d
---
 src/com/android/mail/ui/MaterialSearchActionView.java | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/com/android/mail/ui/MaterialSearchActionView.java b/src/com/android/mail/ui/MaterialSearchActionView.java
index 7b248ce84..50a6f3274 100644
--- a/src/com/android/mail/ui/MaterialSearchActionView.java
+++ b/src/com/android/mail/ui/MaterialSearchActionView.java
@@ -222,8 +222,6 @@ public class MaterialSearchActionView extends LinearLayout implements TextWatche
             } else {
                 mController.onVoiceSearch();
             }
-        } else if (view == mQueryText) {
-            mController.showSearchActionBar(MaterialSearchViewController.SEARCH_VIEW_STATE_VISIBLE);
         }
     }
 
-- 
2.17.1

